test/test_torch.py

```diff
@@ -18181,6 +18181,57 @@ def test_large_linspace(self, device, dtype):
         x = torch.linspace(start, end, steps, dtype=dtype, device=device)
         self.assertGreater(x[1] - x[0], (end - start) / steps)
 
+    @dtypes(torch.int64, torch.float, torch.complex128)
+    def test_moveaxis_invalid(self, device, dtype):
+        shape = self._rand_shape(4, min_size=5, max_size=10)
+        x = self._generate_input(shape, dtype, device, False)
+
+        # Invalid `src` and `dst` dimension
+        with self.assertRaisesRegex(IndexError, "Dimension out of range"):
+            torch.moveaxis(x, 5, 0)
+
+        with self.assertRaisesRegex(IndexError, "Dimension out of range"):
+            torch.moveaxis(x, 0, 5)
+
+        # Mismatch in size of `src` and `dst`
+        with self.assertRaisesRegex(RuntimeError, "moveaxis: Invalid source or destination dims:"):
+            torch.moveaxis(x, (1, 0), (0, ))
+
+        with self.assertRaisesRegex(RuntimeError, "moveaxis: repeated dim in `src`"):
+            torch.moveaxis(x, (0, 0), (0, 1))
+
+        with self.assertRaisesRegex(RuntimeError, "moveaxis: repeated dim in `dst`"):
+            torch.moveaxis(x, (0, 1), (1, 1))
+
+    @unittest.skipIf(not TEST_NUMPY, "NumPy not found")
+    @dtypes(torch.int64, torch.float, torch.complex128)
+    def test_moveaxis(self, device, dtype):
```

Comment:
movedim/moveaxis returns a view. This should also add a test to `TestViewOps` in `TestTorch`. 

@albanD, anything else we should be doing for new view operations?

Reply:
Have added a test in `TestViewOps`

Reply:
You add it to the list [here](https://github.com/pytorch/pytorch/blob/ce443def01a1fef94b17606b95a9ba4574e1323d/tools/autograd/gen_autograd.py#L79) as well just to make sure that it matches with the signature you have and prevent the jit from being confused.

Reply:
Done. Thanks